src/main/java/com/zendesk/maxwell/MaxwellReplicator.java

```diff
@@ -93,11 +93,12 @@ public void work() throws Exception {
 		if (row == null)
 			return;
 
-		if (!skipRow(row)) {
+		if ( !bootstrapper.shouldSkip(row) && !isMaxwellRow(row) ) {
 			producer.push(row);
+		} else {
+			bootstrapper.work(row, producer, this);
```

Comment:
This depends on the fact that the maxwell.bootstrap is writing to the same binlogs as Maxwell is listening to.

There is work in https://github.com/zendesk/maxwell/pull/222 that is making it possible to move the maxwell.schemas table to another mysql host. @osheroff, does that also move all the other maxwell tables to the other host? If so, then these two pull requests would conflict with each other.